<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextPropertyGridWidget</class>
 <widget class="QWidget" name="TextPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>910</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="localizationKeyWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="localizationKeyNameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Localization Key:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="localizationKeyNameLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="localizationValueWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="localizationKeyTextLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Localization Value for current language:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="localizationKeyTextLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="multilineWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="multilineCheckBox">
           <property name="text">
            <string>Multiline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="multilineBySymbolCheckBox">
           <property name="text">
            <string>Multiline by Symbol</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="textWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel">
           <property name="text">
            <string>Text:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fontPresetWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fontPresetLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Font Preset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fontPresetComboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="fontPresetEditButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fontWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fontNameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Font:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fontSelectButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {text-align:right; font-size: 11px; border-style: outset; border: 1px solid #969696; padding: 5px; border-radius: 4px; background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e3e3e3, stop: 1 #f9f9f9);} 
QPushButton:pressed {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #7e7e7e, stop: 1 #f9f9f9); }</string>
           </property>
           <property name="text">
            <string>default font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fontSizeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fontColorWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fontColorLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Font Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QColorWidget" name="textColorWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>184</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>184</width>
             <height>21</height>
            </size>
           </property>
           <property name="text" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fontColorInheritTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Color Inherit Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="colorInheritTypeCombobox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fontPerPixelAccuracyTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="perPixelAccuracyLabel">
           <property name="text">
            <string>Per Pixel Accuracy Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="perPixelAccuracyTypeCombobox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="shadowOffsetWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="shadowOffsetLabel">
           <property name="text">
            <string>Shadow offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="offsetXLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="shadowOffsetXSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="offsetYLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="shadowOffsetYSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="shadowColorGroupWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="shadowColorLabel">
           <property name="text">
            <string>Shadow Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QColorWidget" name="shadowColorWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>184</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>184</width>
             <height>22</height>
            </size>
           </property>
           <property name="text" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="alignWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="AlignLabel">
           <property name="text">
            <string>Align:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="alignComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="useRtlAlignWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="useRtlAlignCheckBox">
           <property name="text">
            <string>Use RTL align</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fittingWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fittingLabel">
           <property name="text">
            <string>Fitting:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fittingTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="isPasswordWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="isPasswordCheckbox">
           <property name="text">
            <string>Is Password</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="maxTextLengthWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="maxTextLengthLabel">
           <property name="text">
            <string>Max Text Length</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxTextLengthSpinBox">
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="autoCapitalizationTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="autoCapitalizationTypeLabel">
           <property name="text">
            <string>Auto Capitalization Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="autoCapitalizationTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="autoCorrectionTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="autoCorrectionTypeLabel">
           <property name="text">
            <string>Auto Correction Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="autoCorrectionTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="spellCheckingTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="spellCheckingTypeLabel">
           <property name="text">
            <string>Spell Check. Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="spellCheckingTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="keyboardAppearanceWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="keyboardAppearanceTypeLabel">
           <property name="text">
            <string>Keyb. Appearance Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="keyboardAppearanceTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="keyboardTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="keyboardTypeLabel">
           <property name="text">
            <string>Keyboard Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="keyboardTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="returnKeyTypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="returnKeyTypeLabel">
           <property name="text">
            <string>Return key type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="returnKeyTypeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="isReturnKeyAutoWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="isReturnKeyAutomatically">
           <property name="text">
            <string>Enable Return Key Automatically</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="UIMarginsPropertyGridWidget" name="marginsWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorWidget</class>
   <extends>QWidget</extends>
   <header>qcolorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>UIMarginsPropertyGridWidget</class>
   <extends>QWidget</extends>
   <header>uimarginspropertygridwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>localizationKeyNameLineEdit</tabstop>
  <tabstop>localizationKeyTextLineEdit</tabstop>
  <tabstop>multilineCheckBox</tabstop>
  <tabstop>multilineBySymbolCheckBox</tabstop>
  <tabstop>textLineEdit</tabstop>
  <tabstop>fontPresetComboBox</tabstop>
  <tabstop>fontPresetEditButton</tabstop>
  <tabstop>fontSelectButton</tabstop>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>colorInheritTypeCombobox</tabstop>
  <tabstop>shadowOffsetXSpinBox</tabstop>
  <tabstop>shadowOffsetYSpinBox</tabstop>
  <tabstop>alignComboBox</tabstop>
  <tabstop>fittingTypeComboBox</tabstop>
  <tabstop>isPasswordCheckbox</tabstop>
  <tabstop>autoCapitalizationTypeComboBox</tabstop>
  <tabstop>autoCorrectionTypeComboBox</tabstop>
  <tabstop>spellCheckingTypeComboBox</tabstop>
  <tabstop>keyboardAppearanceTypeComboBox</tabstop>
  <tabstop>keyboardTypeComboBox</tabstop>
  <tabstop>returnKeyTypeComboBox</tabstop>
  <tabstop>isReturnKeyAutomatically</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
